modules/local/bam_rename.nf

```diff
@@ -0,0 +1,17 @@
+// Import generic module functions
+include { saveFiles; getProcessName } from './functions'
+
+process BAM_RENAME {
+    tag "$meta.id"
+
+    input:
+    tuple val(meta), path(bam)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: sortbam_quant
```

Comment:
```suggestion
    tuple val(meta), path("*.bam"), emit: sortbam_quant
```

a more informative output channel might be good here (I guess copy paste error)

Reply:
I have changed this to bam